examples/bridge/src/bin/dealer.rs

```diff
@@ -29,7 +29,7 @@ fn main() {
         )
         .get_matches();
 
-    // Parse args
+    // Parse arguments
```

Comment:
Would you mind updating the first comment (that already has `Parse arguments` to `Setup parsing`)?